libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/utils/MapUtils.java

```diff
@@ -34,4 +34,23 @@ public static void addLayerToMap(@NonNull Style style, @NonNull Layer layer,
       style.addLayerBelow(layer, idBelowLayer);
     }
   }
+
+  /**
+   * Generic method for adding a new layer to the map that's above another specific layer.
+   *
+   * @param style that the current mapView is using
+   * @param layer a layer that will be added to the map
+   * @param idAboveLayer optionally providing the layer which the new layer should be placed above
+   */
+  public static void addLayerToMapAbove(@NonNull Style style, @NonNull Layer layer,
```

Comment:
Since you're modifying this class it's a good opportunity to write some unit tests.

Reply:
✔️ Written 👍